pkg/api/types.go

```diff
@@ -513,7 +513,8 @@ type EnvVar struct {
 	// Required: This must be a C_IDENTIFIER.
 	Name string `json:"name"`
 	// Optional: no more than one of the following may be specified.
-	// Optional: Defaults to "".
+	// Optional: Defaults to ""; variable references ("$(VAR_NAME)") are
```

Comment:
Like @a-robinson , I would prefer that the pertinent documentation piece goes in the description field for the value.  Also note if you change these you will need to regen the swagger docs.